src/EnergyPlus/ExhaustAirSystemManager.cc

```diff
@@ -0,0 +1,966 @@
+        auto &ip = state.dataInputProcessing->inputProcessor;
+        auto const instances = ip->epJSON.find(cCurrentModuleObject);
+        if (instances != ip->epJSON.end()) {
+            auto const &objectSchemaProps = ip->getObjectSchemaProps(state, cCurrentModuleObject);
+            auto &instancesValue = instances.value();
+            int numZoneExhaustControls = instancesValue.size();
+            int exhCtrlNum = 0;
+
+            if (numZoneExhaustControls > 0) {
+                state.dataZoneEquip->ZoneExhaustControlSystem.allocate(numZoneExhaustControls);
+            }
+
+            for (auto instance = instancesValue.begin(); instance != instancesValue.end(); ++instance) {
+                ++exhCtrlNum;
+                auto const &objectFields = instance.value();
+                auto &thisExhCtrl = state.dataZoneEquip->ZoneExhaustControlSystem(exhCtrlNum);
+                thisExhCtrl.Name = UtilityRoutines::MakeUPPERCase(instance.key());
+                ip->markObjectAsUsed(cCurrentModuleObject, instance.key());
+
+                std::string availSchName = ip->getAlphaFieldValue(objectFields, objectSchemaProps, "availability_schedule_name");
+                int availSchNum = 0;
+                availSchNum = ScheduleManager::GetScheduleIndex(state, availSchName);
+                // UtilityRoutines::FindItemInList(availSchName, state.dataSystemAvailabilityManager->SchedSysAvailMgrData);
+
+                if (availSchNum > 0) {
+                    // normal conditions
+                } else if (availSchNum == 0) {
+                    // blank or anything like that, treat as always available
+                } else {
+                    availSchNum = 0;
+                    // a regular warning is ok.
+                    ShowWarningError(state, RoutineName + cCurrentModuleObject + "=" + thisExhCtrl.Name);
+                    ShowContinueError(state, "Avaiability Manager Name = " + availSchName + "not found.");
+                    // ErrorsFound = true;
+                }
+                thisExhCtrl.AvailScheduleNum = availSchNum;
+
+                // 2022-01-28: an extra zone name field here:
+                std::string zoneName = ip->getAlphaFieldValue(objectFields, objectSchemaProps, "zone_name");
+                thisExhCtrl.ZoneName = zoneName;
+                int zoneNum = UtilityRoutines::FindItemInList(zoneName, state.dataHeatBal->Zone);
+                thisExhCtrl.ZoneNum = zoneNum;
+
+                thisExhCtrl.ControlledZoneNum = UtilityRoutines::FindItemInList(zoneName, state.dataHeatBal->Zone);
+
+                // These two nodes are required inputs:
+                std::string inletNodeName = ip->getAlphaFieldValue(objectFields, objectSchemaProps, "inlet_node_name");
+                // 2022-01: What about nodelist, can GetOnlySingleNode() still used for that?
+                int inletNodeNum =
+                    EnergyPlus::NodeInputManager::GetOnlySingleNode(state,
```

Comment:
You are always within `EnergyPlus` namespace so no need to add it to scope resolution.

Reply:
EnergyPlus:: is now removed at multiple occurrences.